src/server.ts

```diff
@@ -1,23 +1,78 @@
 import { AxiosResponse } from "axios";
-import express, { Express } from "express";
+import express from "express";
  
 const expressApp = express();
 const port = 8000;
-const axios = require('axios');
+const axios = require("axios");
```

Comment:
`require`/`import` is a whole can of worms, but for now just stick to `import`

You can usually make a good guess at how to do so:

`import axios from "axios"`